template.rb

```diff
@@ -175,7 +177,7 @@ def get_gh_file_content(filename)
 gsub_file "bin/setup", "# system('bin/yarn')", "system('bin/yarn')"
 
 # make bin/setup move sample files to new locations
-inject_into_file "bin/setup", after: "system('bin/yarn')\n" do <<-'RUBY'
+inject_into_file "bin/setup", after: %r{system!?.'bin/yarn'.?\n} do <<-'RUBY'
```

Comment:
🥦 Fixed the regex, perhaps the default bin/setup changed.
But this regex should be backward compatible.

Reply:
hmm werid, so the bin/setup now uses `system!` instead of `system`? if so we would need to change the line above too that uncomments the `bin/yarn` system call, it won't find it if no it uses `!`

Reply:
maybe we can do `\('bin/yarn'\)` for the parenthesis? or the new line is `system! 'bin/yarn'`?

what rails version did you test this with?

Reply:
just tested it, it's `system! 'bin/yarn'` now, so let's keep that regexp :+1:

Reply:
and it looks like the previous line that uncomments the system call is not needed anymore either, so let's leave that there in case somebody uses this with an older rails version

Reply:
Sorry late to these comment, but it sounds like it is all good to leave as it?

Reply:
yep!